fastestimator/op/numpyop/univariate/pad_sequence.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from typing import Any, Callable, Dict, Iterable, List, Union
+
+import numpy as np
+
+from fastestimator.op.numpyop.numpyop import NumpyOp
+
+
+class PadSequence(NumpyOp):
+    """Pad sequences to the same length with provided value.
+
+    Args:
+        max_len: Maximum length of all sequences.
+        value: Padding value.
+        padding: Pad before or after the sequences.
```

Comment:
this variable might be better as a boolean called 'append' or something. If it's a string flag then the doc string should enumerate the acceptable values.